apps/nn_ops/common.cpp

```diff
@@ -24,3 +24,13 @@ Expr rounding_shift_right(Expr x, Expr shift) {
 Expr multiply_quantized_multiplier(Expr x, Expr q, Expr shift) {
     return rounding_shift_right(saturating_rounding_doubling_high_multiply(x, q), shift);
 }
+
+int natural_vector_size_with_hexagon(Target target) {
```

Comment:
IMO it would be preferable to just do this where this is used. The meaning of this is ambiguous in a pipeline with Hexagon enabled, but code is still running on the CPU (that's why we don't have a built-in that does something similar... it's tricky to make something that is not confusing).

Reply:
Done.